JS/Lesson24/saffet/filter.js

```diff
@@ -261,10 +270,26 @@ const productsExercise6 = [
 	},
 ];
 
+const outOfStock = productsExercise6.filter(product => {
+	if(product.quantity === 0){
```

Comment:
You need to keep only production that are in stock ( = "filter out the products that are out of stock") so that means products with quality more than 0
```suggestion
	if(product.quantity > 0){
```

Reply:
"out of stock" stokta olmayan ürünler olarak anladım :(